js/interface-lists.js

```diff
@@ -509,12 +529,48 @@ function attahObservers() {
       });
   });
 
+  function highlightError(fieldIds) {
+      _.each(fieldIds, function(id) {
+        $('[data-field-id="' + id + '"] .text-danger').removeClass('hidden');
+      });
+    }
+
+    function checkImageFoldersSelection() {
```

Comment:
The function doesn't just check for image folder selection, but also highlights the error. Overall, I'd say the action is to validate image folder selection.

```suggestion
    function validateImageFoldersSelection() {
```

Reply:
Fixed.